open-hackathon-server/src/hackathon/hack/host_server_manager.py

```diff
@@ -460,15 +460,16 @@ def __get_sms_object(self, hackathon_id):
         :return: ServiceManagementService object
         :rtype: class 'azure.servicemanagement.servicemanagementservice.ServiceManagementService'
         """
-        hackathon_azure_key = self.db.find_first_object_by(HackathonAzureKey, hackathon_id=hackathon_id)
+        hackathon_azure_keys = Hackathon.objects(id=hackathon_id).first().azure_keys
 
-        if hackathon_azure_key is None:
+        if hackathon_azure_keys is None:
             self.log.error('Found no azure key with Hackathon:%d' % hackathon_id)
             return None
 
-        sms = ServiceManagementService(hackathon_azure_key.azure_key.subscription_id,
-                                       hackathon_azure_key.azure_key.get_local_pem_url(),
-                                       host=hackathon_azure_key.azure_key.management_host)
+        hackathon_azure_key = hackathon_azure_keys[0]
+        sms = ServiceManagementService(hackathon_azure_key.subscription_id,
```

Comment:
We don't want to any class under folder `azureformation`, please find a proper class in folder `hazure`